File: OAIP/LR-1-2/4.4.py
X = float(input('Введите заработок за 38 часов работы (в рублях): '))
A = float(input('Введите общее количество отработанных часов: '))
if A > 38:
    regular = 38
    overtime = A - 38
else:
    regular = A
    overtime = 0

total_summa = (regular / 38) * X + overtime * (X / 38) * 1.5
print('Общая сумма заработка: {:.2f} рублей')

File: OAIP/LR-1-2/4.4.pas
program rabota;

var
  X, A: real;
  regular, overtime, total_summa: real;

begin
  writeln('Введите заработок за 38 часов работы (в рублях): ');
  readln(X);
  writeln('Введите общее количество отработанных часов: ');
  readln(A);
  if A > 38 then
  begin
    regular := 38;
    overtime := A - 38;
  end;
  if A <= 38 then
  begin
    regular:= A;
    overtime:= 0;
  end;
  total_summa := (regular / 38) * X + overtime * (X / 38) * 1.5;
  writeln('Общая сумма заработка: ', total_summa:0:2, ' рублей');
end.